.gitmodules

```diff
@@ -11,3 +11,6 @@
 [submodule "external/navigation"]
```

Comment:
you dont need this anymore because its now in requirements.txt

Reply:
this model has simplified collisions. Instead of stl file for collision it's bounding box collision